theano/gpuarray/basic_ops.py

```diff
@@ -293,6 +293,37 @@ class GpuKernelBase(object):
     """
     params_type = gpu_context_type
 
+    def get_params(self, node):
+        # Default implementation, suitable for most sub-classes.
+        # To be necessarly overridden if a subclass uses a ParamsType
+        # (see GpuCumOp, GpuAdvancedIncSubtensor1_dev20 for examples).
```

Comment:
Since we are focusing on the convolution Ops first, and we may not merge the changes to `GpuCumOp` and `GpuAdvancedIncSubtensor1_dev20` yet, it would make sense to update the comment for the moment.